Automattic/woocommerce-payments
# Client side routing to improve the navigation times for "View Deposit History"

Fixes #6527 

#### Changes proposed in this Pull Request

- Used the`getHistory` hook to leverage the client-side navigation. 
- Create a common function that tracks the events and navigate to the `/payments/deposits` page.









#### Testing instructions





- Verify that on clicking the "View deposits history" button, it navigates via the client side and doesn't reload the whole page while navigating.

-------------------


**Post merge**
